frontend/src/modules/ui/LoginSignupModal/LoginSignupModal.js

```diff
@@ -106,8 +107,11 @@ const LoginSignupModal = ({ isVisible }) => {
                             animate={{ x: 0 }}
                             transition={{ stiffness: 120 }}
                         >
+                            <ImageOverlayText>
+                                Find Your Travel Buddy
+                            </ImageOverlayText>
                             <Image
-                                src="/images/src/kanav.webp"
+                                src="/images/fallback/AuthModal.jpg"
```

Comment:
It's recommended to use webp or svg images only.

Reply:
We'll add it later it needs to be replaced with `<ImgwithFallback/>` component and this jpg will be converted to webp